packages/math-input/src/utils.ts

```diff
@@ -46,9 +65,10 @@ const TIMES_ONLY = ["fr", "tr", "pt-pt"];
  * @param {boolean} convertDotToTimes - the setting set by content creators
  * @returns {boolean} - true to convert to × (TIMES), false to use · (CDOT)
  */
-export function convertDotToTimesByLocale(convertDotToTimes: boolean): boolean {
-    const locale = getLocale();
-
+export function convertDotToTimesByLocale(
+    convertDotToTimes: boolean,
+    locale: string,
+): boolean {
```

Comment:
I have a loose preference for `locale` showing up first because it feels most important for any function that takes a `locale` (and gives us consistency for where it shows up in other params).

Reply:
Good call - I can switch this!